### PR TITLE
Feature/news posts metadata

### PR DESCRIPTION
**Pull request by hunter2814**
adds news post metadata description as well as changes regular description to teaser to differentiate the two. Allows meta data to be picked up for posts and hides it when a user is taken to a post.

### CODE REVIEW

File: reason_4.0/lib/core/content_managers/minisite_page.php3 [hunk 1]
```diff
@@ -119,7 +119,7 @@ class MinisitePageManager extends parent_childManager
```
Comment (slylth) [inline reason_4.0/lib/core/content_managers/minisite_page.php3]: I don't think this should be included.

File: reason_4.0/lib/core/content_managers/news.php3 [hunk 2]
```diff
@@ -34,11 +46,14 @@
 			$this -> set_display_name ('release_title', 'Title');
 			$this -> set_display_name ('datetime', 'Date');
 			$this -> set_display_name ('show_hide', 'Show or Hide?');
-			if($this->_is_element('enable_comment_notification')) $this -> set_display_name ('enable_comment_notification', 'Email me when new comments are added to this news item:');
```
Comment (slylth) [inline reason_4.0/lib/core/content_managers/news.php3]: Let's remove this.

**Comment by slylth** [comment]
@mryand The updater included in this PR adds a meta_description to the meta table. Though this seems  like a logical place for meta_description, that field is added to all types that use the meta table. Do you think it should: 
- remain in meta and handled (most likely removed) from other content managers
- be changed to a field specific to news/posts
- in a new table specific to seo needs
- other

**Comment by mryand** [comment]
Composing a type out of multiple tables, and reusing those tables across types, is a style that was used heavily in early Reason development. However, for a variety of reasons (including situations like this one, and the less-than-awesome performance impacts of having to do many joins) we now discourage any further reliance on this type of data structure.

The best practice, now, is to have each type use a single table that is used only for that type. While there are still types that use the legacy style (like News/Posts), when we add new fields to those types we try to follow the current best practices as much as possible by adding the fields to an existing type-specific table. In this case I'd add the field to the press_release table.

I have a goal (which we should probably make an issue) of denormalizing all old-style types to make everything work in roughly the same way.

**Pull request merged**